lib/std/json/test.zig

```diff
@@ -28,7 +28,9 @@ fn testLowLevelScanner(s: []const u8) !void {
     }
 }
 fn testHighLevelDynamicParser(s: []const u8) !void {
-    var parsed = try parseFromSlice(Value, testing.allocator, s, .{});
+    var parsed = try parseFromSlice(Value, testing.allocator, s, .{
+        .duplicate_field_behavior = .use_first,
```

Comment:
is this necessary?

Reply:
Yes, because of [these](https://github.com/ziglang/zig/blob/3e9ab6aa7b2d90c25cb906d425a148abf9da3dcb/lib/std/json/JSONTestSuite_test.zig#L772-L777) tests. Duplicate field names are still valid JSON grammer and MUST therefore be successfully parsed according to RFC 8259. See https://github.com/nst/JSONTestSuite/issues/56